.github/workflows/lint.yaml

```diff
@@ -0,0 +1,34 @@
+name: Lint
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      main
+  schedule:
+    - cron: '28 20 2 * *'
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Toolchain
+        uses: dtolnay/rust-toolchain@v1
+        with:
+          components: clippy, rustfmt
+          toolchain: stable
+
+      - name: Cache
+        uses: Swatinem/rust-cache@v2
+
+      - name: Format
+        run: cargo fmt --all -- --check
+
+      - name: Clippy
+        uses: actions-rs/clippy-check@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          args: --all-targets
```

Comment:
This appears to be a second `clippy` check. Is that intentional?

Reply:
Not sure what you mean. Second to what?

Reply:
Hmm... I thought https://github.com/brave/kuchikiki/pull/5/files#diff-4b122024a3a28ded65da76a2f1bface1f3a27328374438d1298d25585fe0603bR21 was an initial `clippy` check, but I think I'm mistaken.

I thought there might be a double-check because I saw `clippy` twice at https://github.com/brave/kuchikiki/actions/runs/8914366941/job/24529283160.

![image](https://github.com/brave/kuchikiki/assets/43425812/c07009aa-4f4d-45a7-bc85-7af93a37ee34)

Reply:
I think it might run once for the PR and again on the merge, but there are no code changes in a fast-forward merge, so they accumulate?

Reply:
Ah, that might be it.